ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -1745,6 +1926,42 @@ static vk_device ggml_vk_get_device(size_t idx) {
         ggml_vk_create_queue(device, device->compute_queue, compute_queue_family_index, 0, { vk::PipelineStageFlagBits::eComputeShader | vk::PipelineStageFlagBits::eTransfer }, false);
 
         // Shaders
+        // Disable matmul tile sizes early if not supported
```

Comment:
I guess this is replacing the old vendor "guess" functions, but what does "if not supported" mean? Is this still needed or does the shared memory calculation replace it?

Reply:
"Not performant" would be more correct here. The large tiles only run well on Nvidia. The medium tiles only run well on AMD and Nvidia. Intel can only run the small tiles at any decent speed. I think this is related to registers spilling to global memory or overloading caches.